src/app-layout/drawer/index.tsx

```diff
@@ -42,90 +24,143 @@ export interface DesktopDrawerProps {
 // * https://github.com/nvaccess/nvda/issues/5825
 // * https://github.com/nvaccess/nvda/issues/5247
 // * https://github.com/nvaccess/nvda/pull/8869 (reverted PR that was going to fix it)
+export const Drawer = React.forwardRef(
```

Comment:
Even internal components need to have more or less isolated API to easier and safer refactorings in the future.
Suggested API for this component

```js
<Drawer
  ...old props unchanged
  
   ariaLabels={{mainLabel, closeLabel, openLabel}} // this property changes shape
   resizeHandle={...} // new slot
/>
```

Usage examples


```js
// tools (non-resizable) drawer
<Drawer
   ariaLabels={{mainLabel: ariaLabels.tools, openLabel: ariaLabels.toolsToggle, closeLabel: ariaLabels.toolsClose }}
/>

// new (resizable) drawer
<Drawer
   ariaLabels={{mainLabel: ariaLabels.tools, openLabel: ariaLabels.toolsToggle, closeLabel: ariaLabels.toolsClose }}
/>
```